recipe/meta.yaml

```diff
@@ -26,13 +26,17 @@ test:
     - clyent
 
 about:
-  home: https://github.com/Anaconda-Server/clyent
+  home: https://github.com/Anaconda-Platform/clyent
   license: BSD
```

Comment:
We should probably add license version here (e.g. `BSD 3-Clause`). Also is there a license file we can include?